iwyu_include_picker.h

```diff
@@ -226,6 +233,11 @@ class IncludePicker {
   string MaybeGetIncludeNameAsWritten(const string& includer_filepath,
                                       const string& includee_filepath) const;
 
+  // Given a collection of MappedIncludes, and a path that might include them,
+  // choose the best quoted include form for each MappedInclude.
+  vector<string> RelativiseMappedIncludesForPath(
```

Comment:
`...ForIncluder`?

The method name is a little hard to read, probably because of `Relativise`... I wonder if something like `BestQuotedIncludeForIncluder` would be better? But that's also a mouthful. I'd like to move from "quoted include" to "include name" at some point, but I don't think this is a good time to start.

Reply:
Went with `BestQuotedIncludesForIncluder`.